### PR TITLE
First pass at renaming

### PR DESCRIPTION
**Pull request by tmaul**
* added FBNEO_DEBUG flag. This automatically includes FBA_DEBUG if enabled just in case we ever port anything from the rotten corpse of FB Alpha
* Changed the build target name
* Changed a few refrences to FB Alpha that should say FinalBurn Neo

There are obviously more changes to make, but it's a start ;)

### CODE REVIEW

File: src/burn/drv/pre90s/d_galaga.cpp [hunk 1]
```diff
@@ -2088,21 +2088,8 @@ static struct N54XX_Sample_Info_Def galagaN54xxSampleList[] =
...
-	.tilemapsConfig         = galagaTilemapConfig,
-	.drawLayerTable         = galagaDrawFuncs,
-	.drawTableSize          = GALAGA_DRAW_TBL_SIZE,
-	.getSpriteParams        = galagaGetSpriteParams,
-	.reset                  = galagaReset,
-	.customRWTable          = galagaCustomICRW,
-	.n54xxSampleList        = galagaN54xxSampleList
+	galagaCPU,galagaWriteTable,galagaReadTable,galagaMemTable,GALAGA_MEM_TBL_SIZE,galagaROMTable, GALAGA_ROM_TBL_SIZE,0x2000,
+	galagaTilemapConfig, galagaDrawFuncs, GALAGA_DRAW_TBL_SIZE, galagaGetSpriteParams,galagaReset,galagaCustomICRW,galagaN54xxSampleList
 };
```
Comment (dinkc64) [inline src/burn/drv/pre90s/d_galaga.cpp]: Can you make this look better at least?
f.ex:
{
  /*cpus*/ *tab* galagaCPU,
  /*wrAddrList*/ *tab* galagaWriteTable,

I can do it if you don't want to :)

Comment (tmaul) [reply src/burn/drv/pre90s/d_galaga.cpp]: I sure can :)

Comment (tmaul) [reply src/burn/drv/pre90s/d_galaga.cpp]: Ah, saw you did it already :)

Comment (dinkc64) [reply src/burn/drv/pre90s/d_galaga.cpp]: oops, can ya please remove d_galaga from your PR, then I can merge it

Comment (tmaul) [reply src/burn/drv/pre90s/d_galaga.cpp]: should be ok now

**Pull request merged**